crown-court-contribution/src/main/java/uk/gov/justice/laa/crime/contribution/service/ContributionService.java

```diff
@@ -203,4 +205,14 @@ public boolean hasApplicationStatusChanged(RepOrderDTO repOrderDTO, CaseType cas
                 && repOrderDTO.getRorsStatus() != null
                 && !repOrderDTO.getRorsStatus().equalsIgnoreCase(status);
     }
+
+    public boolean hasContributionBeenSent(final int repId, final String laaTransactionId) {
+        List<Contribution> contribList = maatCourtDataService.findContribution(repId, laaTransactionId, Boolean.FALSE);
+        List<Contribution> contributionList = Optional.ofNullable(contribList).orElse(Collections.emptyList()).stream().filter(
+                contribution ->   ("SENT".equals(contribution.getTransferStatus()) &&
```

Comment:
I would consider breaking out the filter into 2 separate filters, as I feel that multiple filters with very simple conditions are easier to understand than one filter with compound logic. i.e.

```
                .filter(contribution ->   ("SENT".equals(contribution.getTransferStatus())))
                .filter(contribution -> contribution.getMonthlyContributions().compareTo(BigDecimal.ZERO) > 0)
```

instead of:

```
.filter(
                contribution ->   ("SENT".equals(contribution.getTransferStatus()) &&
                                            contribution.getMonthlyContributions().compareTo(BigDecimal.ZERO) > 0)
```